plugin/index.php

```diff
@@ -1,14 +1,18 @@
 <?php
-
 /**
  * Plugin Name: Gutenberg
  * Plugin URI: https://wordpress.github.io/gutenberg/
  * Description: Prototyping since 1440. Development plugin for the editor focus in core.
  * Version: 0.1
  */
-
+// Hook.
```

Comment:
>My intention of adding this comment is always to be liberal with documentation.

There's an underlying assumption here the "more documentation is always better", to which my comment is primarily directed at disputing.